src/engine/loopingcontrol.cpp

```diff
@@ -122,84 +122,84 @@ LoopingControl::LoopingControl(QString group,
     // DEPRECATED: Use beatloop_size and beatloop_set instead.
     // Activates a beatloop of a specified number of beats.
     m_pCOBeatLoop = new ControlObject(ConfigKey(group, "beatloop"), false);
-    connect(m_pCOBeatLoop, SIGNAL(valueChanged(double)), this,
-            SLOT(slotBeatLoop(double)), Qt::DirectConnection);
+    connect(m_pCOBeatLoop, &ControlObject::valueChanged, this,
+            [=](double value){slotBeatLoop(value);}, Qt::DirectConnection);
```

Comment:
This one is still a lambda

Reply:
This is required because the slot takes more (default) parameters than the signal emits.